src/screens/Home/components/TransactionsList/index.tsx

```diff
@@ -35,20 +35,22 @@ const TransactionsList = (props: TransactionsListProps) => {
     [styles.blankDivider],
   );
 
-  const stickyHeaderIndices = sections
-    .map((item, index) => {
-      if (typeof item === 'string') {
-        return index;
-      }
-      return null;
-    })
-    .filter((item) => item !== null) as number[];
+  // Disable the sticky header since there is a bug in FlashList that cause
+  // the first element to not update correctly when the date changes.
```

Comment:
```suggestion
  // Disable the sticky header since there is a bug in FlashList that causes
  // the first element to not update correctly when the date changes.
```